libs/embed/src/embed.ts

```diff
@@ -148,7 +144,11 @@ class Novu {
       } else {
         hideWidget();
       }
-    });
+    }
+
+    window.addEventListener('resize', positionIframe);
+    window.addEventListener('click', handleClick);
+    window.addEventListener('touchstart', handleClick);
```

Comment:
Use the same listener to `click and touchstart` events. This fixes the issue on the touch devices that it doesn't react to the touch events.